### DISCUSSION TITLE
-flatten makes multi tiff preview combined

### DISCUSSION THREAD

**Discussion by kasimertan** [ImageMagick/ImageMagick#5038] [Help]
Hello,
Opentext Media Management uses imagemagick for previewing.
We have added below command to create thumbnails without background of image;
-intent relative -resize 1024x768 -quality 99 -colorspace sRGB -strip -auto-orient -flatten

But this is creating half of the each image of tiff  file as a preview.
We want to show only first image as a preview.
How can we manage that?

Thank you.

**Comment by snibgo** [comment]
What version of IM, on what platform?

Please show the complete command, and a sample input file.

**Reply by kasimertan** [reply]
We have solved our issue with below command.
Thank you for your interest.

`-background white -alpha background -alpha off -bordercolor white -border 1 -alpha off -compose copy_opacity -composite -shave 1x1 -intent relative -resize 1024x768 -quality 95 -colorspace sRGB -strip -auto-orient`
